tests/trig.rs

```diff
@@ -345,26 +343,51 @@ mod test {
         Limits,
     };
 
-    #[test]
-    fn og_recipe() {
-        let complex: Ruleset<Trig> = Ruleset::from_file("scripts/trig/complex.rules");
-        let limits = Limits {
-            iter: 3,
-            node: 2000000,
-        };
+    // Extra rules about `cis` and `I` to "fast-forward" rule synthesis
+    fn prior_forbidden_rules() -> Ruleset<Trig> {
+        Ruleset::new([
+            // constant folding for PI
+            "(+ PI PI) ==> (* 2 PI)",
```

Comment:
so the constant folding ones shouldn't be forbidden, right? They don't have `cis` or `I`, no?

Reply:
Renamed